changelog/unreleased/change-adopt-oc-table-files

```diff
@@ -0,0 +1,6 @@
+Change: Adopt oc-table-files from ods
```

Comment:
Technically not a change but an enhancement/internal refactoring since it doesn't change anything for admins/users of OC web

Reply:
goo point, i updated it, thanks